src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -6079,4 +6079,7 @@ Oluşturma sırasında NuGet'in paketleri indirmesini önlemek için, Visual Stu
   <data name="PushCommandTimeoutError" xml:space="preserve">
     <value>Pushing took too long. You can change the default timeout of 300 seconds by using the -Timeout &lt;seconds&gt; option with the push command.</value>
   </data>
+  <data name="Error_AssemblyInformationalVersion" xml:space="preserve">
+    <value>Invalid AssemblyInformationalVersion {0}.</value>
```

Comment:
Is there any more info that can be given? Like which assembly is it, or where is the AssemblyInformationalVersion specified? Just want to make sure that people can figure out what to fix when they see this error.